test/BaselineOutput/Common/LdSvm/netcoreapp/LDSVM-nob-CV-breast-cancer-out.txt

```diff
@@ -11,43 +11,43 @@ Confusion table
           ||======================
 PREDICTED || positive | negative | Recall
 TRUTH     ||======================
- positive ||      127 |        7 | 0.9478
- negative ||        9 |      211 | 0.9591
+ positive ||      123 |       11 | 0.9179
+ negative ||       17 |      203 | 0.9227
           ||======================
-Precision ||   0.9338 |   0.9679 |
-OVERALL 0/1 ACCURACY: 0.954802
-LOG LOSS/instance:  0.251296
+Precision ||   0.8786 |   0.9486 |
+OVERALL 0/1 ACCURACY: 0.920904
```

Comment:
Less accuracy here. Do we know where the minor differences are compounding from?

Reply:
The biggest thing is just that the dataset we are using for these tests is tiny, so any minor change we make to anything will affect the accuracy. The tests I ran on actual full size datasets didn't really show any changes in accuracy.